content/en/account_management/billing/pricing.md

```diff
@@ -1,26 +1,56 @@
 ---
 title: Pricing
-kind: faq
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "https://www.datadoghq.com/pricing"
+  tag: "Pricing"
+  text: "Datadog Pricing"
 ---
 
 ## Overview
 
 Datadog has many pricing plans to fit your needs. Visit the [Pricing][1] page for more details.
 
-### Hourly
+## Calculating pricing
 
-Datadog supports hourly pricing. The standard hourly rate for Datadog Pro is $0.03 per host, per hour.
+Unless otherwise stated in your Order, Datadog calculates fees based on product usage during each calendar month. Here are the most common pricing units:
 
-Choose to pay for all of your hosts hourly, or commit to a subset of hosts on a monthly or annual plan and pay any additional hosts on an hourly basis (billed at the end of each month). This could be less expensive for extra hosts that are up for a short time, but more than monthly rates if the hosts are up for extended periods of time.
+### Infrastructure monitoring
 
-Contact [Sales][2] or your [Customer Success][3] Manager to discuss hourly pricing for your account.
+- A **host** is a physical or virtual operating system instance.  Datadog records the number of hosts you are concurrently monitoring in the Datadog Infrastructure service once an hour. 
+   - On a high watermark plan (HWMP), these hourly measurements are ordered from highest to lowest at the end of the month, and Datadog charges based on the eighth highest measurement.  
+  - On a hybrid monthly/hourly plan (MHP), Datadog charges your minimum monthly commitment, and for any host hours above that commitment, Datadog charges an hourly rate.
+- A **container** is a self-contained operating environment that includes application software and limited operating system libraries and settings.  Datadog records the number of containers you are concurrently monitoring in the Datadog Infrastructure service at five-minute intervals.  Datadog charges monthly based on the fractional hours of monitored containers.
+- A **custom metric** is a single, unique combination of a metric name, host ID and any tags.  Datadog charges based on the monthly average of unique custom metrics submitted to the Datadog Infrastructure service per hour.
```

Comment:
Should we link here to the dedicated custom metrics doc?

Reply:
yes! good idea